database/migrations/2019_02_25_171041_add_jedi_fields.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddJediFields extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('dues_packages', function (Blueprint $table) {
+            $table->timestamp('access_start')->after('effective_end')->useCurrent();
+            $table->timestamp('access_end')->after('access_start')->useCurrent();
+        });
+
+        Schema::table('teams', function (Blueprint $table) {
+            $table->unsignedInteger('project_manager')->nullable()->comment('user_id of the project manager');
```

Comment:
Please make this `project_manager_id` to avoid potential conflict with relationship names on the model.